R/mpt.R

```diff
@@ -46,18 +61,28 @@ mpt <- function(formula, data, tree, model,
                 default_prior_intercept = "normal(0, 1)",
                 default_prior_coef = "normal(0, 0.5)",
                 default_priors = TRUE,
+                data_format = "long",
+                log_p = F,
```

Comment:
Can you change `F` to `FALSE` (also throughout the code if it exists elsewhere)?